packages/shared/__tests__/stylex-transform-value.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+import transformValue from '../src/transform-value';
+
+describe('transformValue content property tests', () => {
+  test('preserves CSS functions without quotes', () => {
+    const functions = [
+      ['counters(div, ".")', 'counters(div, ".")'],
+      ['counter(chapter)', 'counter(chapter)'],
+      ['counter(chapter, upper-roman)', 'counter(chapter, upper-roman)'],
+      ['attr(href)', 'attr(href)'],
+      ['url(image.jpg)', 'url(image.jpg)'],
+      [
+        'linear-gradient(#e66465, #9198e5)',
+        'linear-gradient(#e66465, #9198e5)',
+      ],
+      [
+        'image-set("image1x.png" 1x, "image2x.png" 2x)',
+        'image-set("image1x.png" 1x, "image2x.png" 2x)',
+      ],
+    ];
+
+    functions.forEach(([input, expected]) => {
+      expect(transformValue('content', input, {})).toBe(expected);
+    });
+  });
+
+  test('preserves CSS keywords without quotes', () => {
+    const keywords = [
+      'normal',
+      'none',
+      'open-quote',
+      'close-quote',
+      'no-open-quote',
+      'no-close-quote',
+      'inherit',
+      'initial',
+      'revert',
+      'revert-layer',
+      'unset',
+    ];
+
+    keywords.forEach((keyword) => {
+      expect(transformValue('content', keyword, {})).toBe(keyword);
+    });
+  });
+
+  test('handles mixed content values', () => {
+    const mixedValues = [
```

Comment:
Same here. Values are left unchanged, so you can use an array of strings rather than an array of pairs.